assignments/admin.py

```diff
@@ -1,18 +1,51 @@
 from django.contrib import admin
-from assignments.models import Assignment, Section, OpenTextTask
-from nested_admin.nested import NestedModelAdminMixin, NestedStackedInline, NestedTabularInline
+from assignments.models import (Assignment, Section, OpenTextTask, BudgetTextTask, BudgetTextTarget, MapObject,
+                                BudgetMapTask)
+from nested_admin.nested import NestedModelAdminMixin, NestedTabularInline
 from leaflet.admin import LeafletGeoAdmin
+from django import forms
+
+
+class BudgetForm(forms.ModelForm):
+    def clean_range(self):
+        '''
+        Include upper boundary if boundary value is set
+        :return: numericrange object with upper boundary included
+        :rtype: NumericRange
+        '''
+        if self.cleaned_data['range'] is not None:
+            self.cleaned_data['range']._bounds = '[]'
```

Comment:
Good point, tried to avoid it, but I couldn't find it. The problem is that for models FloatRangeField used here, form field that is created is forms.FloatRangeField. For that field on field's clean method input value is converted to python NumericRange type in compress method where NumericRange is instantiated with input values. As NumericRange type default bounds are '[)' which means that upper bound is not taken into consideration, and  I had to fix it somehow, because user would expect that both bounds are valid values. There was a way to create new field type with overwritten  compress method entirely or that way I used to change cleaned data on forms clean. I am not sure how to do it in a more cleaner manner.